_posts/2023-07-10-trino-fest-2023-redis.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "Redis & Trino - Real-time indexed SQL queries (new connector)"
+author: "Allen Terleto, Julien Ruaux, Ryan Duan"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/Redis.png
+show_pagenav: false
+---
+
+Ever since the pandemic, the world has changed. The need for a digital first
+economy rose dramatically during the lockdowns. It has become clear that
+digital's impact on future businesses will be more and more important. As Redis'
+Field CTO Allen Terleto said during their talk from [Trino Fest 2023]({%
+post_url 2023-06-20-trino-fest-2023-recap %}), "In a digital first economy, data
+is the lifeblood of the organization, which makes the databases the heart of
+enterprise architectures". Redis, one of the most popular open source projects,
```

Comment:
I don't think "one of the most popular" means much. Just explain what it is, and if you want to hype it up, say, "it's **a** popular open source project."